dashboard/app/models/lesson_group.rb

```diff
@@ -125,11 +125,11 @@ def localized_display_name
   end
 
   def localized_description
-    I18n.t("data.script.name.#{script.name}.lesson_groups.#{key}.description", default: nil)
+    I18n.t("data.script.name.#{script.name}.lesson_groups.#{key}.description", default: description)
```

Comment:
Have you tested this for a brand new script where you save the DSL after creating new lesson groups? I ran into some weird stuff in the past where defaults caused issues here

Reply:
I will try this out and report back.

Reply:
tangentially related issue -- when I check out staging branch and try to create a new script, I see something like this in the initial DSL:
![Screen Shot 2020-11-13 at 3 32 23 PM](https://user-images.githubusercontent.com/8001765/99130579-97c53180-25c5-11eb-85a4-7b5488137a7d.png)

Then, when I try to save, I get this error:
![Screen Shot 2020-11-13 at 3 32 12 PM](https://user-images.githubusercontent.com/8001765/99130591-a1e73000-25c5-11eb-9ba6-946de67d3ff1.png)

it seems like we should be providing a default display name for lessons when creating a new script. does that sound right?

levelbuilders may very well always have had to edit the dsl before they could save, for example because we required a lesson to have one lesson in it until recently. so maybe this is not a recent regression, but could be a nice quick improvement in the legacy script edit case.

Reply:
To address your request, when I change the script DSL for a new script to this:
```
lesson_group 'legacy-lesson-group', display_name: 'legacy display name'
lesson 'new-lesson', display_name: 'New Lesson'

lesson_group 'other-lesson-group', display_name: 'other display name'
lesson 'other-lesson', display_name: 'Other Lesson'
```
then everything looks right to me after I save.

![Screen Shot 2020-11-13 at 3 41 05 PM](https://user-images.githubusercontent.com/8001765/99130968-b546cb00-25c6-11eb-86f6-5c29570880bb.png)

![Screen Shot 2020-11-13 at 3 41 42 PM](https://user-images.githubusercontent.com/8001765/99130996-c5f74100-25c6-11eb-9338-9939a6aa551e.png)

(sorry it is named legacy-staging, I did this out on my branch and not staging)

Reply:
If I create another new script, and then immediately change the url to use the beta script editor, I see:
![Screen Shot 2020-11-13 at 3 44 24 PM](https://user-images.githubusercontent.com/8001765/99131249-82510700-25c7-11eb-8fa7-3de93dcc4f02.png)

when I save, it succeeds, and I see a mostly empty script overview page. 

From this point on, if I go and add any combination of Lessons and Lesson Groups using the beta script editor page, i get what I expect to see on the script overview page.

Reply:
@dmcavoy Please let me know if you think this covers the issue you were asking about.

Reply:
Yes this addresses it. Sorry for not getting back to you sooner.